orcestra_book/plans/HALO-20240903a.md

```diff
@@ -110,16 +110,16 @@ waypoints = [
      ec_north.assign(fl=410),
      c_mid.assign(fl=410),
      x_xsd.assign(fl=410),
-     IntoCircle(c_south.assign(fl=410), radius, -360),   
-     IntoCircle(c_mid.assign(fl=430), radius, -360),
-     c_mid.assign(fl=450),
+     IntoCircle(c_south.assign(fl=410), radius, -360),
+     x_cms.assign(fl=450),
+     IntoCircle(c_mid.assign(fl=450), radius, -360),
```

Comment:
This changes the flight level of the circle, which was not mentioned in the PR's description. I want to cross-check if this was intentional or by error.

Reply:
thanks for catching this.  It is indeed a change, originally we were at 430 as 450 is often not possible so soon, but it is on the edge so I thought to try to get that high sooner just in case.